security/login_link.rst

```diff
@@ -0,0 +1,510 @@
+.. index::
+   single: Security; Login link
+   single: Security; Magic link login
+
+How to use Passwordless Login Link Authentication
+=================================================
+
+Login links, also called "magic links", allow users to login without
+passwords. Whenever a user wants to login, a login link is generated and
+send to the user (e.g. using an e-mail). The user is logged in as soon as
+they visit that link.
+
+Login links are supported by Symfony when using the experimental
+authenticator system. You must
+:ref:`enable the authenticator system <security-enable-authenticator-manager>`
+in your configuration to use this feature.
+
+Using the Login Link Authenticator
+----------------------------------
+
+This guide assumes you have set-up security and have created a user object
+in your application. Follow :doc:`the main security guide </security>` if
+this is not yet the case.
+
+1) Configure the Login Link Authenticator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The login link authenticator is configured using the ``login_link`` option
+under the firewall. In the most minimal form, you have to configure a
+``check_route``:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/security.yaml
+        security:
+            firewalls:
+                main:
+                    login_link:
+                        check_route: login_check
```

Comment:
You also will now need `signature_properties` - after https://github.com/symfony/symfony/pull/38528